platform/base.go

```diff
@@ -45,14 +45,15 @@ type BaseCluster struct {
 
 	name       string
```

Comment:
Could also switch this from `string` to `platform.Name`

Reply:
That's the cluster name, not the platform name.  The intent of `platform.Name` is that it should be treated as an enumerated type.